### PR TITLE
Master move repo arm timer

### PR DESCRIPTION
**Pull request by claudiasima-ro**

### CODE REVIEW

File: plugins/org.polarsys.capella.scenario.editor.dsl.ide/xtend-gen/org/polarsys/capella/scenario/editor/dsl/ide/TextualScenarioIdeSetup.java [hunk 1]
```diff
@@ -1,15 +1,3 @@
...
- *  This program and the accompanying materials are made available under the
- *  terms of the Eclipse Public License 2.0 which is available at
- *  http://www.eclipse.org/legal/epl-2.0
- *  
- *  SPDX-License-Identifier: EPL-2.0
- *  
- *  Contributors:
- *     Thales - initial API and implementation
- ******************************************************************************/
 /**
```
Comment (georgiana-ecobici) [inline plugins/org.polarsys.capella.scenario.editor.dsl.ide/xtend-gen/org/polarsys/capella/scenario/editor/dsl/ide/TextualScenarioIdeSetup.java:1]: No copyright.

File: plugins/org.polarsys.capella.scenario.editor.dsl.ide/xtend-gen/org/polarsys/capella/scenario/editor/dsl/ide/TextualScenarioIdeModule.java [hunk 2]
```diff
@@ -1,15 +1,3 @@
...
- *  This program and the accompanying materials are made available under the
- *  terms of the Eclipse Public License 2.0 which is available at
- *  http://www.eclipse.org/legal/epl-2.0
- *  
- *  SPDX-License-Identifier: EPL-2.0
- *  
- *  Contributors:
- *     Thales - initial API and implementation
- ******************************************************************************/
 /**
```
Comment (georgiana-ecobici) [inline plugins/org.polarsys.capella.scenario.editor.dsl.ide/xtend-gen/org/polarsys/capella/scenario/editor/dsl/ide/TextualScenarioIdeModule.java:1]: No copyright.

File: plugins/org.polarsys.capella.scenario.editor.dsl.ui/xtend-gen/org/polarsys/capella/scenario/editor/dsl/ui/TextualScenarioUiModule.java [hunk 3]
```diff
@@ -1,15 +1,3 @@
...
- *  This program and the accompanying materials are made available under the
- *  terms of the Eclipse Public License 2.0 which is available at
- *  http://www.eclipse.org/legal/epl-2.0
- *  
- *  SPDX-License-Identifier: EPL-2.0
- *  
- *  Contributors:
- *     Thales - initial API and implementation
- ******************************************************************************/
 /**
```
Comment (georgiana-ecobici) [inline plugins/org.polarsys.capella.scenario.editor.dsl.ui/xtend-gen/org/polarsys/capella/scenario/editor/dsl/ui/TextualScenarioUiModule.java:1]: No copyright.

File: plugins/org.polarsys.capella.scenario.editor.dsl/src-gen/org/polarsys/capella/scenario/editor/dsl/textualScenario/Role.java [hunk 4]
```diff
@@ -8,7 +8,7 @@
  *  SPDX-License-Identifier: EPL-2.0
  *  
  *  Contributors:
- *     Thales - initial API and implementation
+ *       Thales - initial API and implementation
  ******************************************************************************/
 /**
```
Comment (georgiana-ecobici) [inline plugins/org.polarsys.capella.scenario.editor.dsl/src-gen/org/polarsys/capella/scenario/editor/dsl/textualScenario/Role.java]: I think if the header you applied had 2 " " removed, this won't show diff on so many files.

File: plugins/org.polarsys.capella.scenario.editor.dsl/src/org/polarsys/capella/scenario/editor/dsl/provider/TextualScenarioProvider.java [hunk 5]
```diff
@@ -1,46 +1,46 @@
...
-    resourceSet.getResources().add(resource);
-    return resource;
-  }
-
-  public XtextResource getResource() {
-    return resource;
-  }
-}
+/*******************************************************************************
+ * Copyright (c) 2020 THALES GLOBAL SERVICES.
```
Comment (georgiana-ecobici) [inline plugins/org.polarsys.capella.scenario.editor.dsl/src/org/polarsys/capella/scenario/editor/dsl/provider/TextualScenarioProvider.java]: This files seems unchanged, shall not appear on diff.

File: plugins/org.polarsys.capella.scenario.editor.dsl.ide/xtend-gen/org/polarsys/capella/scenario/editor/dsl/ide/TextualScenarioIdeModule.java (hunk 2 above)
Comment (claudiasima-ro) [reply plugins/org.polarsys.capella.scenario.editor.dsl.ide/xtend-gen/org/polarsys/capella/scenario/editor/dsl/ide/TextualScenarioIdeModule.java:1]: There were 2 copyright headers, I removed one.

File: plugins/org.polarsys.capella.scenario.editor.dsl.ide/xtend-gen/org/polarsys/capella/scenario/editor/dsl/ide/TextualScenarioIdeSetup.java (hunk 1 above)
Comment (claudiasima-ro) [reply plugins/org.polarsys.capella.scenario.editor.dsl.ide/xtend-gen/org/polarsys/capella/scenario/editor/dsl/ide/TextualScenarioIdeSetup.java:1]: There were 2 copyright headers, I removed one and left only one.

File: plugins/org.polarsys.capella.scenario.editor.dsl.ui/xtend-gen/org/polarsys/capella/scenario/editor/dsl/ui/TextualScenarioUiModule.java (hunk 3 above)
Comment (claudiasima-ro) [reply plugins/org.polarsys.capella.scenario.editor.dsl.ui/xtend-gen/org/polarsys/capella/scenario/editor/dsl/ui/TextualScenarioUiModule.java:1]: There were 2 copyright headers, I removed one of them.

File: plugins/org.polarsys.capella.scenario.editor.dsl/src-gen/org/polarsys/capella/scenario/editor/dsl/textualScenario/Role.java (hunk 4 above)
Comment (claudiasima-ro) [reply plugins/org.polarsys.capella.scenario.editor.dsl/src-gen/org/polarsys/capella/scenario/editor/dsl/textualScenario/Role.java]: Done.

**Pull request closed without merging**